README.md

```diff
@@ -52,7 +52,8 @@ to provide transparency in the provenance of all data.
 | `$BUCKET/co/`  | Cloud Optimized | A port of gaussian-gridded ERA5 data to Zarr.                                 |
 | `$BUCKET/raw/` | Raw Data        | All raw grib & NetCDF data.                                                   |  
 
-Files are updated from ECMWF on a monthly cadence (on roughly the 9th of each month) with a 3 month delay, which avoids including preliminary versions of ERA5. The date of the latest available data can be found by inspecting the "time" axis of each Zarr store.
+ - Files are updated from ECMWF on a **monthly cadence** (on roughly the 9th of each month) with a 3 month delay, which avoids including preliminary versions of ERA5.
+ - The most recent data available can be found by examining the metadata associated with each Zarr store. The metadata encompasses three essential attributes: `valid_time_start`, `valid_time_end`, and `last_updated`. These attributes specify the start date, end date, and most recent time of update for the dataset's data, respectively.
```

Comment:
Nit: I like the name `valid_time_stop` rather than `valid_time_end` because it matches the start/stop convention of Python's `slice` object.

Please note that both start and stop times are inclusive, and all times are given in UTC.

Reply:
Done.